backend/api/v1/endpoints/research.py

```
@@ -400,3 +400,23 @@ def accept_collaboration(
     except Exception as e:
         logging.error(f"Error accepting collaboration request: {str(e)}")
         raise HTTPException(status_code=500, detail= give_error)
+
+@router.get("/papers/user/{user_id}")
+def get_papers_by_user(user_id: int, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_papers_by_user`

(ANN201)